### PR TITLE
Remove duplicate codes

### PR DESCRIPTION
**Pull request by saketkumar95**
Was going through the code and got this some.

### CODE REVIEW

File: src/StylesProvider.js [hunk 1]
```diff
@@ -8,7 +8,6 @@ import themeLight from './styles/themeLight';
 
 const themeNameToObject = {
   'default': themeLight,
-  'light': themeLight,
```
Comment (Sam1301) [inline src/StylesProvider.js]: @saketkumar95 consider this commit https://github.com/zulip/zulip-mobile/commit/a21bf82446594450217a850b73e42882aae21aa2, through probably this change shouldn't cause a problem with the production app...

Comment (saketkumar95) [inline src/StylesProvider.js]: I didn't get that commit. The app is working fine after removing this. Tested it on both iOS and android.

Comment (borisyankov) [inline src/StylesProvider.js]: It is intentional. We used to call this theme 'light' and if this line is removed, it might produce `undefined`.

Comment (saketkumar95) [inline src/StylesProvider.js]: oh, Weird! 
BDW Updated 
Thanks @borisyankov @Sam1301 for this info.

**Pull request merged**